docs/services.html

```diff
@@ -0,0 +1,121 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Great Idea!</title>
+
+  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
+  <link rel="stylesheet" href="@/../css/index.css">
+  <link rel="stylesheet" href="@/../css/services.css">
+
+  <!--[if lt IE 9]>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+  <![endif]-->
+</head>
+
+<body>
+<!-- Header Area -->
+<header>
+  <nav>
+    <a href="@/../services.html">Services</a>
+    <a href="#">Product</a>
+    <a href="#">Vision</a>
+    <a href="#">About</a>
+    <a href="#">Contact</a>
+  </nav>
+  <a class="logo" href="@/../index.html"><img class="logo" src="img/logo.png" alt="Great Idea! Company logo."></a>
+</header>
+<!-- End Header Area -->
+<div class="container">
+<!-- Hero Section -->
+    <hero class="short-hero">
+			<img class="services-header-img" src="img/services-header.jpg" alt="Our services header image">
+			<div class="hero-content">	
+				<h2>Services</h2>
+				<p>Our services provide the best digital value in nulla suscipit interdum diam at maximus. Proin vitae tellus
+				rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus
+				et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque eleifend. Mauris euismod
+				facilisis iaculis.</p>
+			</div>
+    <!-- End Hero Section -->
+	</hero>
+	<div class="divider-line"><br></div>
+    <!-- Main Content Section -->
+	<section class="main-content">
+		<card>
```

Comment:
Oof. Although there are ways to create your own HTML elements with frameworks like React and Vue, there are a few reasons you would _not_ want to do this in vanilla HTML:

- They are not allowed by the HTML specification
- They might conflict with future standard elements with the same name
- There is probably an existing HTML element that is better suited to the task

So, you could opt instead to use an element like [article](https://devdocs.io/html/element/article) or [section](https://devdocs.io/html/element/section) here.